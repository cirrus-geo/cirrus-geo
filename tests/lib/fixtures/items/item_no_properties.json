{
    "type": "Feature",
    "stac_version": "1.0.0-beta.2",
    "stac_extensions": ["eo", "view", "proj"],
    "id": "S2B_17HQD_20201103_0_L2A",
    "bbox": [
      -78.85549714480591, -33.5090673116811, -77.66654434530413,
      -33.171471085391424
    ],
    "geometry": {
      "type": "Polygon",
      "coordinates": [
        [
          [-78.84720324626629, -33.5090673116811],
          [-78.85549714480591, -33.171471085391424],
          [-78.68855580607664, -33.20224951876674],
          [-78.68565102210097, -33.20281229571297],
          [-78.43490780996153, -33.25398334754271],
          [-78.12962160400237, -33.31843449933885],
          [-77.92995605454519, -33.36103653713505],
          [-77.66899441916789, -33.41891691608237],
          [-77.66654434530413, -33.482909622076725],
          [-78.84720324626629, -33.5090673116811]
        ]
      ]
    },
    "collection": "sentinel-s2-l2a",
    "assets": {},
    "links": {}
  }
