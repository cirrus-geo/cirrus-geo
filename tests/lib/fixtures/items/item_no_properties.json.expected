{
  "collection": {
    "DataType": "String",
    "StringValue": "sentinel-s2-l2a"
  },
  "bbox.ll_lon": {
    "DataType": "Number",
    "StringValue": "-78.85549714480591"
  },
  "bbox.ll_lat": {
    "DataType": "Number",
    "StringValue": "-33.5090673116811"
  },
  "bbox.ur_lon": {
    "DataType": "Number",
    "StringValue": "-77.66654434530413"
  },
  "bbox.ur_lat": {
    "DataType": "Number",
    "StringValue": "-33.171471085391424"
  }
}
